### ISSUE TITLE
Accessibility with this Issues web page

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#344]
```
What steps will reproduce the problem?
1. Within Windows colour settings change colours to give white text on a black 
background. Do not select a high contrast theme.
2. Look at the buttons on this page and the buttons available when entering an 
issue.
3.

What is the expected output? What do you see instead?
Text and background colour of the buttons is the same and therefore the text is 
not visible. This issue is explained in this WAI accessibility guideline:
http://www.w3.org/TR/WCAG-TECHS/F24.html
Both foreground and background colours should be set. One of these has not been 
set on the web page and is defaulting to the OS colour setting. This means on 
my system both colours end up the same.
(The Google account creation page also has this problem. The main Google search 
site had it too but was fixed when I contacted the team)

What version of the product are you using? On what operating system?
Happens with all brouwser except Chrome which seems to fix this error on the 
fly. I'm using Firefox 26.0

Please provide any additional information below.



```

Original issue reported on code.google.com by `mossieg...@gmail.com` on 14 Jan 2014 at 3:45

**Comment by GoogleCodeExporter** [comment]
```
Oops. Soory for the word order in the summary. It's very late and I'm very 
tired. and now I have to face the dreaded Captch again :(
```

Original comment by `mossieg...@gmail.com` on 14 Jan 2014 at 3:47

**Comment by GoogleCodeExporter** [comment]
```
Attachedis a screen shot showing the appearance of the bottom on one of thse 
bug reportys. Note the buttons on the bottom left. The text is no visible. I 
have my OS colours set to white text on a black background. The background 
colour of the buttons has been set by the web page to white. But the text 
colour has not been set and so defaults to OS colour (white). Hence, white on  
white. Both foreground and background colours should be set. Or neither should 
be set allowing fallback to OS defaults for both. Setting one but not the other 
causes a lot of problems. This is a problem right through the Google web sites. 
It mostly manifests in text entry boxes, but also buttons as here.

Actually the Captcha text box has the same problem :(
```

Original comment by `mossieg...@gmail.com` on 15 Jan 2014 at 2:24

Attachments:
- [Eye-Free-buttons.png](https://storage.googleapis.com/google-code-attachments/eyes-free/issue-344/comment-2/Eye-Free-buttons.png)
